docs/appendix/zowe-glossary.md

```diff
@@ -0,0 +1,188 @@
+
+* **ZLUX**  
+
+  This is a codename for the Zowe Application Framework, consisting of the Zowe desktop and App server. We should replace as many occurrences of "ZLUX" in our documentation with "desktop", "app framework" and "app-server" according to which part we are talking about. More mentions of ZLUX lead to more confusion. 
+
+* **API Gateway**  
+
+  The API Gateway is a proxy server that routes requests from clients on its northbound edge, such as web browsers or the Zowe command line interface, to servers on its southbound edge that are able to provide data to serve the request. The API Gateway is also responsible for generating the authentication token used to provide single sign-on (SSO) functionality. See https://docs.zowe.org/stable/getting-started/zowe-architecture/#api-gateway
+
+* **API Catalog** 
+
+  The API Catalog provides a list of the API services that have registered themselves as catalog tiles. 
+
+* **API Discovery** 
+
+  a service to allow RESTful services to be dynamically added to the API Catalog
+
+* **App Server** 
+
+  The App Server is a node.js server that is responsible for the Zowe Application Framework. This server provides the Zowe desktop, which is accessible through a web browser. The Zowe desktop includes a number of applications that run inside the Application Framework such as a 3270 emulator and a File Editor.     
+
+* **Caching Service** 
+
+  This feature is designed for Zowe components in a high availability configuration. It supports the High Availability of all components within Zowe, allowing components to be stateless by providing a mechanism to offload their state to a location accessible by all instances of the service, including those which just started 
+       
+* **Zowe Cross Memory Server (X-MEM)** - V1: 
+  
+  is an APF authorized server application that provides privileged services to Zowe in a secure manner. The Cross Memory server is a low-level privileged server for managing mainframe data securely. For security reasons, it is not an HTTP server. Instead, this server has a trust relationship with ZSS. Other Zowe components can work through ZSS in order to handle z/OS data that would otherwise be unavailable or insecure to access from higher-level languages and software.    
+
+* **z/OS Explorer Services**
+  
+* **Zowe Systems Services Server (ZSS)** 
+
+  The Zowe desktop delegates a number of its services to the ZSS server which it accesses through the http port 8542. ZSS is written in C and has native calls to z/OS to provide its services. For more information, see Zowe architecture | Zowe Docs
+      
+* **ZIS (Zowe Interprocess Services)**  - V2: 
+
+  it is a server but not a webserver.    
+
+* **Zowe server infrastructure**  
+
+  The glue that binds the servers together (zowe-install-packaging)       
```

Comment:
The set of programs (ex "zwe") and utilities (ex JCL, scripts) which manage the Zowe server configuration and components.
The infrastructure standardizes the packaging of components and controls how they are started, stopped, and how configuration is provided to them.